src/c++/vdo/base/block-map.c

```diff
@@ -2906,7 +2906,7 @@ int vdo_decode_block_map(struct block_map_state_2_0 state,
 	int result;
 	zone_count_t zone = 0;
 
-	STATIC_ASSERT(VDO_BLOCK_MAP_ENTRIES_PER_PAGE ==
+	BUILD_BUG_ON(VDO_BLOCK_MAP_ENTRIES_PER_PAGE !=
```

Comment:
FOr multi-line statements like this, make sure the following lines are indented correctly.  In this case, the next two lines are indented one space too far to the right.